web/js/containers/map-interactions.js

```diff
@@ -104,7 +105,8 @@ const mapDispatchToProps = dispatch => ({
         CompletelyCustomModal: vectorDialog,
         isResizable: true,
         dragHandle: '.modal-header',
-        dialogKey: new Date().getUTCMilliseconds(),
+        dialogKey,
```

Comment:
Looks like we are passing `dialogKey` twice as both `dialogKey` and `key`.  Probably just need one of these?

Reply:
Ahh didn't see this comment until already merged. `dialogKey` is something being used by the child. But the `key` is not really a prop and isn't accessible to the child. Changing the key causes a the parent react-component to re-render (I didn't learn this until semi-recently, super useful)

Reply:
Ah right, it's _that_ `key